server/database/schema.sql

```diff
@@ -1,21 +1,312 @@
+
+
+CREATE TABLE user (
+   id INT PRIMARY KEY AUTO_INCREMENT,
+   firstname VARCHAR(50) NOT NULL,
+   lastname VARCHAR(50) NOT NULL,
+   role VARCHAR(50) NOT NULL,
+   email VARCHAR(50) NOT NULL,
+   password VARCHAR(30) NOT NULL
+);
+
+INSERT INTO user (firstname, lastname, role, email, password) 
+   VALUES
+      ('Léa', 'Benoit', 'customer', 'lea.benoit@example.com', 'azerty123'),
+      ('Thomas', 'Marchand', 'customer', 'thomas.marchand@example.com', 'azerty123'),
+      ('Emma', 'Robert', 'customer', 'emma.robert@example.com', 'azerty123'),
+      ('Lucas', 'Fernandez', 'customer', 'lucas.fernandez@example.com', 'azerty123'),
+      ('Chloé', 'Perrin', 'customer', 'chloe.perrin@example.com', 'azerty123'),
+      ('Nathan', 'Rousseau', 'customer', 'nathan.rousseau@example.com', 'azerty123'),
+      ('Camille', 'Lemoine', 'customer', 'camille.lemoine@example.com', 'azerty123'),
+      ('Enzo', 'Morel', 'customer', 'enzo.morel@example.com', 'azerty123'),
+      ('Manon', 'Lefevre', 'customer', 'manon.lefevre@example.com', 'azerty123'),
+      ('Hugo', 'Faure', 'customer', 'hugo.faure@example.com', 'azerty123'),
+      ('Sophie', 'Garnier', 'customer', 'sophie.garnier@example.com', 'azerty123'),
+      ('Julien', 'Lopez', 'customer', 'julien.lopez@example.com', 'azerty123'),
+      ('Clara', 'Noël', 'customer', 'clara.noel@example.com', 'azerty123'),
+      ('Antoine', 'Blanc', 'customer', 'antoine.blanc@example.com', 'azerty123'),
+      ('Laura', 'Mathieu', 'customer', 'laura.mathieu@example.com', 'azerty123'),
+      ('Mathis', 'Fontaine', 'customer', 'mathis.fontaine@example.com', 'azerty123'),
+      ('Anaïs', 'Meyer', 'customer', 'anais.meyer@example.com', 'azerty123'),
+      ('Hugo', 'Chevalier', 'customer', 'hugo.chevalier@example.com', 'azerty123'),
+      ('Élise', 'Lemoine', 'customer', 'elise.lemoine@example.com', 'azerty123'),
+      ('Axel', 'Barbier', 'customer', 'axel.barbier@example.com', 'azerty123'),
+      ('Jade', 'Guillot', 'customer', 'jade.guillot@example.com', 'azerty123'),
+      ('Noah', 'Giraud', 'customer', 'noah.giraud@example.com', 'azerty123'),
+      ('Louise', 'Gomez', 'customer', 'louise.gomez@example.com', 'azerty123'),
+      ('Victor', 'Clement', 'customer', 'victor.clement@example.com', 'azerty123'),
+      ('Emma', 'Fabre', 'customer', 'emma.fabre@example.com', 'azerty123'),
+      ('Léo', 'Lemoine', 'customer', 'leo.lemoine@example.com', 'azerty123'),
+      ('Zoé', 'Paris', 'customer', 'zoe.paris@example.com', 'azerty123'),
+      ('Gabriel', 'Rolland', 'customer', 'gabriel.rolland@example.com', 'azerty123'),
+      ('Clémentine', 'Adam', 'customer', 'clementine.adam@example.com', 'azerty123'),
+      ('Louis', 'Perrot', 'customer', 'louis.perrot@example.com', 'azerty123');
+
+
+CREATE TABLE event (
+   id INT PRIMARY KEY AUTO_INCREMENT,
+   title VARCHAR(100) NOT NULL,
+   date DATE NOT NULL,
+   event_begining TIME NOT NULL,
```

Comment:
pas besoin de prefixe

Reply:
et suivez mes conseils sur les formats de date

Donc 

start_at DATETIME NOT NULL
end_at DATETIME NOT NULL